### PR TITLE
LoadAVG for Darwin

### PR DESCRIPTION
**Pull request by njuettner**
I'm using the using some functions from linux. There's no pseudo proc that's why I'm using systctl vm.loadavg.

Basically it has the same logic like linux. I also included pty which is a pseudo terminal for unix.

Hopefully you can use it somehow :).

### CODE REVIEW

**Comment by syamp** [comment]
Thanks for the PR! Couple of comments:
 * Generally we avoid spawning commands via inspect where possible. Would it be possible to use sysctl syscall interface instead.  Please checkout cpustat,memstat_darwin for examples.

**Comment by njuettner** [comment]
Thanks for suggestion, I'll take a look and try to change it

File: os/loadstat/loadstat_darwin.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	}()
+	return s
+}
+
+// Collect populates Loadstat by using sysctl
+func (s *LoadStat) Collect() {
+	var loadavg [3]C.double
+
+	C.getloadavg(&loadavg[0], 3)
+	s.OneMinute.Set(misc.ParseFloat(fmt.Sprintf("%.2f", loadavg[0])))
```
Comment (syamp) [inline os/loadstat/loadstat_darwin.go]: You can probably do an explicit cast: s.OneMinute.Set(float64(loadavg[0]))

**Comment by syamp** [comment]
Added a comment and if you are able to address it and sign CLA - I will merge it away (CLA: https://github.com/square/inspect/blob/master/CONTRIBUTING.md)

Thanks again.

**Pull request merged**